src/websocket/common.cc

```diff
@@ -168,6 +172,26 @@ future<> connection::send_data(opcodes opcode, temporary_buffer<char>&& buff) {
         header[1] = uint8_t(buff.size());
     }
 
+    temporary_buffer<char> write_buf;
```

Comment:
Unused in this patch. Maybe it's used later, but then it should as well be added "later"